pyhap/accessory_driver.py

```diff
@@ -1013,3 +996,16 @@ def signal_handler(self, _signal, _frame):
         except Exception as e:
             logger.error("Could not stop AccessoryDriver because of error: %s", e)
             raise
+
+    def _notify(self, queries, client_addr):
```

Comment:
```suggestion
    def _notify(self, queries: List[Dict[str, Any]], client_addr: Tuple[str, int]) -> None:
```

Will need `Tuple` import above

Reply:
Could even be `Iterable` as far as I can see.